client/src/rubin/nublado/client/nubladoclient.py

```diff
@@ -807,22 +807,26 @@ def __init__(
         if logger is None:
             logger = structlog.get_logger()
         self._logger = logger
-
+        self._timeout = timeout
+        self._base_url = base_url
+        self._initialize_client()
         # Construct a connection pool to use for requests to JupyterHub. We
         # have to create a separate connection pool for every user, since
         # each will get user-specific cookies set by JupyterHub. If we shared
         # connection pools, users would overwrite each other's cookies and
         # get authentication failures from labs.
```

Comment:
This comment is now misplaced and should either be a docstring on `_initialize_client` or before the invocation of `_initialize_client`.